### ISSUE TITLE
b-pagination menu keyboard navigation Jaws Support

### ISSUE THREAD

**Issue by fabo15x** [bootstrap-vue/bootstrap-vue#4322]
When using Jaws Screen reader, the menu keyboard navigation is announced as "to move through items press the UP or DOWN arrow" but the supported keys for such navigation in the component are LEFT and RIGHT instead.
The b-pagination component should support UP and DOWN arrow keys.

**Comment by tmorehouse** [comment]
This might be an issue with JAWS, as the list should be interpreted as [`aria-orientation="horizontal"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-orientation_attribute), but maybe we need to explicitly add the `aria-orientation` attribute.

**Comment by tmorehouse** [comment]
@fabo15x PR #4322 adds in support for UP/DOWN arrow keys (in adition to the existing LEF/RIGHT arrow keys) and will be available in release v2.1.0

**Comment by fabo15x** [comment]
> @fabo15x PR #4322 adds in support for UP/DOWN arrow keys (in addition to the existing LEFT/RIGHT arrow keys) and will be available in release v2.1.0

What a quick turnaround. Thank you so much!

**Comment by tmorehouse** [comment]
BootstrapVue `v2.1.0` has been released, which has a fix for this issue.
